subsys/bluetooth/mesh/dfu_srv.c

```diff
@@ -304,8 +304,6 @@ static int handle_start(struct bt_mesh_model *mod, struct bt_mesh_msg_ctx *ctx,
 			status = BT_MESH_DFU_SUCCESS;
 		}
 
-		srv->update.ttl = ttl;
-		srv->blob.state.xfer.id = blob_id;
```

Comment:
If server returns status `BT_MESH_DFU_SUCCESS` why it doesn't update `ttl` and `blob_id`?

Reply:
I think this will not have any effect. I think the second else clause will not execute as I don't think there is a situation where DFU server is busy, but there is no active update going on.

Reply:
Should status be set unconditionally then into `BT_MESH_DFU_ERR_WRONG_PHASE`?